### ISSUE TITLE
install with cuda=9.0

### ISSUE THREAD

**Issue by 13717630148** [rusty1s/pytorch_sparse#183]

**Comment by rusty1s** [comment]
You can still install it based on the CUDA version PyTorch was compiled with (given by `torch.version.cuda`).

**Comment by 13717630148** [comment]
i dont understand.i ran following code and got result as follows:
`print(torch.__version__)
1.5.0+cuda90_cudnn7.6.3_lms
>>> print(torch.version.cuda)
9.0
>>> print(torch.backends.cudnn.version())
7603`
but the wheels available only offer cpu or cuda>=9.2 versions.

**Comment by rusty1s** [comment]
You can try to run:
```
pip install --verbose torch-scatter==2.0.5 torch-sparse==0.6.7
```
Please send the log if you encounter any issues.
